client/src/index.html

```diff
@@ -167,7 +167,10 @@ <h2>Section title</h2>
     <!-- Bootstrap core JavaScript
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
-    <script src="./Dashboard Template for Bootstrap_files/jquery-3.3.1.slim.min.js.download" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script 
+      src="https://code.jquery.com/jquery-3.3.1.min.js"
```

Comment:
You can knock off the https: protocol and just have it be src="//code.jquery.com/jquery-3.3.1.min.js" to allow this to work with both http and https.

Reply:
Having trouble with this one.  It just isn't working.  I remove http: but it cannot find the file.